frontend/src/components/ide/AIAssistant/components/PendingChanges.jsx

```
@@ -0,0 +1,245 @@
+            "border rounded-lg overflow-hidden transition-all",
+            change.status === 'pending' ? "border-yellow-500/30 bg-yellow-500/5" : "",
+            change.status === 'applying' ? "border-blue-500/30 bg-blue-500/5" : "",
+            change.status === 'applied' ? "border-green-500/30 bg-green-500/5" : "",
+            change.status === 'rejected' ? "border-red-500/30 bg-red-500/5 opacity-50" : ""
+        )}>
+            {/* Header */}
+            <div
+                className="flex items-center gap-2 p-3 cursor-pointer hover:bg-muted/30"
+                role="button"
+                tabIndex={0}
+                onClick={() => setIsExpanded(!isExpanded)}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)